src/i18n/translations/pl.json

```
@@ -0,0 +1,70 @@
+{
+  "dash": {
+    "header": {
+      "your_account_heading": "Twoje konto",
+      "account": "Konto",
+      "settings": "Ustawienia",
+      "log_out": "Wyloguj się"
+    },
+    "sidebar": {
+      "logo_by": "od",
+      "tournament": {
+        "catname": "turniej",
+        "overview": "Panel główny",
+        "ladder": "Drabinka turniejowa",
+        "standings": "Rankingi"
+      },
+      "tournament_data": {
+        "catname": "informacje o turnieju",
+        "competing_teams": "Drużyny",
+        "debate_motions": "Tezy debat",
+        "staff_judges_bias": "Organizatorzy i sędziowie",
+        "physical_infrastructure": "Miejsca",
+        "event_branding": "Szata graficzna"
+      },
+      "shareable": {
+        "catname": "materiały prasowe",
+        "image_generation": "Generowanie grafik"
+      },
+      "organisational": {
+        "catname": "organizacja",
+        "audit_log": "Dziennik zdarzeń",
+        "users": "Użytkownicy",
+        "tournament_settings": "Ustawienia turnieju"
+      }
+    }
+  },
+  "login": {
+    "msg_handle_empty": "Handle must not be empty.",
```

```suggestion
    "msg_handle_empty": "Nick nie może być pusty.",
```
String not translated. :P